api/src/pcapi/core/booking_providers/api.py

```diff
@@ -3,6 +3,7 @@
 from pcapi.core.booking_providers.cds.client import CineDigitalServiceAPI
 from pcapi.core.booking_providers.models import BookingProviderClientAPI
 from pcapi.core.booking_providers.models import BookingProviderName
+from pcapi.core.booking_providers.models import Movie
 from pcapi.core.booking_providers.models import SeatMap
 from pcapi.core.booking_providers.models import Ticket
 from pcapi.core.booking_providers.models import VenueBookingProvider
```

Comment:
```suggestion
import pcapi.core.booking_providers.models as booking_providers_models
```
cf commentaire précédent